### PR TITLE
Marcgabe15/readme change

### PR DESCRIPTION
**Pull request by marcgabe15**
# Description
This updated the front end read me

# Related Issue

# Motivation and Context

# How Has This Been Tested?

# Screenshots (if appropriate):

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,42 @@
 # edugator-staff-frontend
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app), using the [Redux](https://redux.js.org/) and [Redux Toolkit](https://redux-toolkit.js.org/) template.
+An interactive coding and learning portal for students in STEM.
+
+Need info? Check the Docs | Create an Issue | Ask us on Discord
```
Comment (kapooramanpreet) [inline README.md:5]: Would be good to add the discord join link somewhere.

**Review by kapooramanpreet — APPROVED** [review]
Rest LGTM.

File: README.md (hunk 1 above)
Comment (marcgabe15) [reply README.md:5]: I'll add to a later PR

Comment (kapooramanpreet) [reply README.md:5]: Sounds good.

**Pull request merged**